qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -200,6 +200,31 @@
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "echo foo > %s/foo"
             mountpoint_def = "/tmp"
+            virt_test_type = qemu
+            images += " data"
+            force_create_image_data = yes
+            force_remove_image_data = yes
+            start_vm = yes
+            storage_pools = default
+            storage_type_default = "directory"
+            storage_pool = default
+            base_tag = "data"
+            image_size_data = 100M
+            image_name_data = data
+            snapshot_tag = sn1
+            image_size_sn1 = 100M
+            image_name_sn1 = sn1
+            image_format_sn1 = qcow2
+            device = "drive_data"
+            format = qcow2
+            rebase_mode = unsafe
+            snapshot_file = sn1
```

Comment:
Remove these three lines?

Reply:
snapshot_file = "sn1" was modified and it was needed by snapshot_test.check_snapshot()